client/src/feedbacker-components/survey-edit/survey-edit-container.js

```diff
@@ -110,14 +116,17 @@ export default class SurveyEditContainer extends React.Component {
       openId: null,
       pendingDelete: null,
       edit: { },
+      disableSlack: false,
```

Comment:
Nothing seems to write to `disableSlack` so it might be always `false` ?

Reply:
componentDidMount does when site is loaded, I don't think there needs to be other cases?